src/View/Widget/MultiCheckboxWidget.php

```diff
@@ -152,6 +153,9 @@ protected function _renderInputs($data, $context)
             if (!isset($checkbox['templateVars'])) {
                 $checkbox['templateVars'] = $data['templateVars'];
             }
+            if (!isset($checkbox['label'])) {
```

Comment:
Ignore me, I'm a muppet. I was misreading the diff and thought this chunk was connected to the chunk above that defaults `label` to `true`.